packages/tokens/README.md

```diff
@@ -5,41 +5,14 @@ Hopper tokens are all the values needed to construct and maintain a design syste
 [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](../../LICENSE)
 [![npm version](https://img.shields.io/npm/v/@hopper-ui/tokens)](https://www.npmjs.com/package/@hopper-ui/tokens)
 
-## Installation
```

Comment:
I wouldn't remove the installation information completely as it may be useful for some devs.

Reply:
My toughts was that we have a user documentation, with the installation process, so we should focus on having one, up to date, source of truth. 

If you think we should still leave the installation process in the packages, i can re-add them if you reply to this comment

Reply:
That's a good point.
The advantage of leaving it is that it will be visible to everyone both in Github and in NPM. Sometimes it's easier to find the info at the source than to go to the documentation :)

Reply:
I'll leave/add the installation part in all npm packages